smarts/core/smarts.py

```diff
@@ -378,6 +378,13 @@ def _setup_bullet_client(self, client: bc.BulletClient):
             with pkg_resources.path(models, "plane.urdf") as path:
                 plane_path = str(path.absolute())
 
+        client.setPhysicsEngineParameter(
```

Comment:
I suggest you can apply these parameters in the previous `setPhysicsEngineParameter()`

https://github.com/huawei-noah/SMARTS/blob/2fddbea00620a3e8a2a2b305e05f136ccf94b47f/smarts/core/smarts.py#L363-L366